tracker-agent/src/main/configuration/cumulocity.json

```diff
@@ -14,6 +14,7 @@
     "ROLE_ALARM_READ",
     "ROLE_ALARM_ADMIN",
     "ROLE_DEVICE_CONTROL_READ",
+    "ROLE_DEVICE_BOOTSTRAP",
```

Comment:
What is this whole cumulocity.json for since this is not run as regular microservice?

This role is required for getting device credentials, isn't it?